Week1/homework/app.js

```diff
@@ -2,10 +2,130 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'harry_potter_chamber_secrets',
+    'desert_solitaire',
+    'geek_love',
+    'gilead',
+    'disgrace',
+    'infinite_jest',
+    'lolita',
+    'maus',
+    'poems',
+    'things_fall_apart'
   ];
+  console.log(bookTitles);
 
+  //2.3 
+
+  /*  // hold this function  based on request 2.5 
+
+  function createAndAppend(titles) {
+    const ul = document.createElement("ul"),
+      root = document.getElementById("root");
+    root.appendChild(ul);
+
+    for (let i = 0; i < titles.length; i++) {
+      const li = document.createElement("li");
+      ul.appendChild(li);
+      li.setAttribute('id', titles[i]);
+    }
+  }
+
+  createAndAppend(bookTitles);
```

Comment:
Put this call in a function `main` that you execute through `window.onload`. This ensures that your code runs when the web page has completed loading.